apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -104,6 +111,16 @@ type ObservabilitySpec struct {
 	Tracing TracingConfigSpec `json:"tracing,omitempty"`
 }
 
+// MetricsConfigSpec defines a metrics config.
+type MetricsConfigSpec struct {
+	// Enabled specifies if a ServiceMonitor should be created for each Tempo component.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Enabled"
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
I wonder if we should be more explicit in the name - e.g. `CreateServiceMonitor` . 

Also do we know if Tempo will stick with p8s metrics? Are there any plans to move to OTLP metrics? cc) @frzifus

Reply:
y, i guess being a bit more explicit would make sense.

> Are there any plans to move to OTLP metrics?

Not that i would know of. I couldnt find a ticket and there is no otel metric instrumentation in their code.

Reply:
Ok, I've updated the field name and displayName.